main.py

```diff
@@ -611,8 +613,17 @@ def modBook():
 
 def editMember():
     def addMember():
+        membername = addMemberFNameE.get() + " " + addMemberLNameE.get()
+        mycursor = mycon.cursor()
+        mycursor.execute(f"UPDATE TABLE CUSTOMERS SET member = 'Y' WHERE Customer Name LIKE '{membername}'AND Contact Number LIKE '{addMemberContactE.get()}'" )
+        mycursor.commit()
+        mycursor.execute(f"SELECT Customer ID FROM CUSTOMERS WHERE Customer Name Like'{membername} AND COntact LIKE'{addMemberContactE.get()}'")
```

Comment:
SELECT custid FROM customers WHERE name LIKE '{membername}' AND contact LIKE '{addMemberContactE.get()}'